Gemfile.lock

```diff
@@ -229,7 +229,7 @@ GEM
     multipart-post (2.1.1)
     netrc (0.11.0)
     nio4r (2.5.8)
-    nokogiri (1.13.3)
+    nokogiri (1.14.2)
```

Comment:
Nice 👍 worth clarifying the commit message to "Update Nokogiri to latest version"?